plaidml/bridge/openvino/ops/read_value.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+
+#include "ngraph/opsets/opset3.hpp"
+
+#include "ngraph/opsets/opset4.hpp"
+
+#include "plaidml/op/op.h"
+
+#include "plaidml/edsl/edsl.h"
+
+#include "plaidml_util.hpp"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+using namespace ngraph;
+
+namespace PlaidMLPlugin {
+
+void registerReadValue() {
+  registerOp("ReadValue", [](const Context& ctx) {
+    auto* layer = ngraph::as_type<ngraph::opset3::ReadValue>(ctx.layer);
+    IE_ASSERT(ctx.operands.size() == 1);
+    auto I = ctx.operands.at(0);
+    I = I + 0;  // Plaidml doesn't support return without operation
```

Comment:
Wonder if you can check variable before do this, and maybe something like "auto A = I" to create a new tensor is better

Reply:
Wonder if we shall check input type and variable too

Reply:
according to the documentation of readvalue, we just need to check type and shape of output.

Reply:
yes, so we shall confirm the input is same with the output request, do you know if they will always be the same?

Reply:
i have tried auto A = I, it didn't work.